zchronod/zchronod/src/bin/client_read.rs

```diff
@@ -20,24 +21,19 @@ fn main() -> std::io::Result<()> {
         data = query_by_key_id(GatewayType::MergeLog);
     }
     
-    let destination = "127.0.0.1:8050";
-    socket.send_to(&data, destination)?;
+    stream.write_all(&data)?;
 
     // recv msg
     let mut buf = [0; 65535];
-    match socket.recv_from(&mut buf) {
-        Ok((size, _)) => {
-            let msg = prost::bytes::Bytes::copy_from_slice(&buf[..size]);
-            let  response= Innermsg::decode(msg).unwrap();
-            let ret = QueryResponse::decode(response.message.unwrap().data.as_ref()).unwrap();
-            println!("Received response: {:?}", ret);
-        }
-        Err(ref err) if err.kind() == std::io::ErrorKind::WouldBlock => {
-            println!("No response received.");
-        }
-        Err(err) => {
-            eprintln!("Error receiving response: {}", err);
-        }
+    let size = stream.read(&mut buf)?;
```

Comment:
You may need to do `stream.read` for multiple times if client send bytes larger than TCP MTU. So we need to define the transport protocol first.

Reply:
will refactor when later